generator-fraudulentlogic/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "generator-fraudulentlogic",
+  "version": "0.1.0",
+  "description": "",
+  "files": [
+    "generators"
+  ],
+  "main": "generators/index.js",
+  "keywords": ["yeoman-generator"],
+  "dependencies": {
```

Comment:
Do we need this package.json at all?

Reply:
I was going through the docs and it says we should have this under the generators folder. https://yeoman.io/authoring/